app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -56,95 +71,122 @@ fun HomeScreen(
     HomeScreenContent(
         showMyCoinsLoading = showMyCoinsLoading,
         showTrendingCoinsLoading = showTrendingCoinsLoading,
+        isRefreshing = rememberRefreshing,
         myCoins = myCoins,
         trendingCoins = trendingCoins,
         onMyCoinsItemClick = viewModel.input::onMyCoinsItemClick,
         onTrendingItemClick = viewModel.input::onTrendingCoinsItemClick
-    )
+    ) { viewModel.loadData(isRefreshing = true) }
```

Comment:
should execute via an input

```suggestion
    ) { viewModel.input.loadData(isRefreshing = true) }
```